sortedm2m/fields.py

```diff
@@ -316,31 +311,9 @@ def get_rel_to_model_and_object_name(self, klass):
             to_object_name = to_model._meta.object_name
         return to_model, to_object_name
 
-    def get_intermediate_model_sort_value_field_default(self, klass):
-        def default_sort_value(name):
```

Comment:
Hi! I'll try my best to explain... First of all, in general, calculating default values shouldn't have to hit the database. Doing so would hide away potential performance issues, and wouldn't really cooperate with Django migrations. It also could be the source of bugs if it depended on the caller knowing that it hits the database and thus ensuring it is called inside the appropriate atomic transaction.

In this case, we were using the `get_default` value only as a starting point for subsequent values (except in the Django 1.5 case where it was hitting the database every time). Now, the reader inline exactly how the starting point is calculated and be sure it's inside the transaction and thus "correct". Also, this important, it technically wouldn't quite work to keep this calculation inside `get_default` because it depends on the source object's primary key (it finds the maximum sort value the exists for that source object and increments from there).

Hope that helps to clear things up, thanks!